assets/locales/en.json

```diff
@@ -116,5 +117,14 @@
   "itemsFailedNotification": {
     "title": "Failed to add items",
     "message": "Could not add your items to your node."
+  },
+  "runGarbageCollectorWarning": {
+    "title": "Garbage collector",
+    "message": "Running the garbage collector will remove all the objects that are not pinned or on your MFS from your repository. This action might take a while. Do you want to run the Garbage Collector?",
```

Comment:
Maybe:
“Running the garbage collector will remove from your repository all objects that are not pinned or on your MFS. This may take a while. Do you wish to proceed?”

Reply:
Perfect!